conf/uno.conf

```diff
@@ -91,6 +91,27 @@ if [[ -n "$FLUO_YARN_REPO" ]]; then
   export FLUO_YARN_TARBALL=fluo-yarn-$FLUO_YARN_VERSION-bin.tar.gz
 fi
 
+# Version compatability checks
+# ----------------------------
+# Apache built Accumuo 1.x only works with Hadoop 2.x.  Apache built Accumulo
+# 2.x only works with Hadoop 3.x.  However if building Accumulo, these
+# constraints may not apply.  The following sanity checks fail fast when
+# versions are incompatible.  If the checks cause a problem, comment them out
+# and consider submitting an issue or PR.
+#
+
+if [[ -z "$ACCUMULO_REPO" ]]; then
+  if [[ "${ACCUMULO_VERSION}" = 1.* && ! "${HADOOP_VERSION}" = 2.* ]]; then
```

Comment:
I presume we're not trying to support versions of Accumulo old enough for it to matter that this is only true for 1.6+?

Reply:
I have no interest in supporting older than 1.7 at this point.  I added a check for hadoop 1.x.